classes/plugins/PubObjectsExportPlugin.inc.php

```diff
@@ -338,6 +409,35 @@ function getAdditionalFieldNames($hookName, $args) {
 	}
 
 	/**
+	 * @copydoc AcronPlugin::parseCronTab()
+	 */
+	function callbackParseCronTab($hookName, $args) {
+		$taskFilesPath =& $args[0];
+		$taskFilesPath[] = $this->getPluginPath() . DIRECTORY_SEPARATOR . 'scheduledTasks.xml';
+		return false;
+	}
+
+	/**
+	 * Retrieve all unregistered articles.
+	 * @param $context Context
+	 * @return array
+	 */
+	function getUnregisteredArticles($context) {
```

Comment:
Generally speaking, it's better (I think) to use iterator objects rather than pulling everything out into an array. Since some of these lists are going to involve a lot of articles, using the iterator will instantiate them one at a time, where converting to an array will need them all to exist in memory at once.